.snippets/text/products/multigov/multigov-timeline.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "title": "[Submit an integration request with Tally](https://www.tally.xyz/get-started)",
+        "content": "Apply for MultiGov access by submitting your request to Tally.",
+        "icon": ":fontawesome-solid-1:"
+    },
+    {
+        "title": "Choose a deployment network",
```

```suggestion
        "title": "Deploy MultiGov to the Hub and Spoke Chains of Your Choice",
```

Sorry I know I requested this be changed already haha, but looking at it, I think this will make the steps flow a bit better